.gitignore

```diff
@@ -100,3 +100,9 @@ docs/*.html
 # IDE related
 .idea/
 .vscode/
+
```

Comment:
Can you remove changes that aren't relevant for this PR? This includes changes to `.gitignore` here as well as deletion of CSVs.